les/server_handler.go

```diff
@@ -939,14 +940,14 @@ func (h *serverHandler) broadcastHeaders() {
 				p := p
 				switch p.announceType {
 				case announceTypeSimple:
-					p.queueSend(func() { p.SendAnnounce(announce) })
+					p.mustQueueSend(func() { p.sendAnnounce(announce) }, p.closeCh)
```

Comment:
Do not block here if queue is full; this will block new head announcements for all clients. Missing a head announcement is fine if there is a congestion (this should happen only in extreme cases anyway).